src/s1ap/handlers/attach_secreq.c

```diff
@@ -132,12 +132,16 @@ get_secreq_protoie_value(struct proto_IE *value)
 	value->data[2].nas.elements = (nas_pdu_elements *)
 			malloc(SEC_MODE_NO_OF_NAS_IES * sizeof(nas_pdu_elements));
 
+	// TODO: it should be ue security capabilities, not network capabilities
+	// for now, only return the 2 mandatory capabilities
```

Comment:
Please edit the comment to mention we are ignoring security with this patch. In case there is a security issue required with the test tools, or when we resume security developments, it will be easy to track.
Recommendation: Instead of number 2, in the same function, you can make it #define UE_SECURITY_NONE OR UE_SECURITY_DISABLED for clarity.

Reply:
The first 2 security capability are EEA and EIA, do not understand why you think the UE_SECURITY is none?

Reply:
I will change the comment and the #define.

Reply:
@vbarate The security part is not disabled. The UE sends security capabilities for 3G and 2G networks also. These are not needed during an LTE attach. So, as of now we only copy and send back the EUTRAN security capabilites.
Recommendation : We can rename the number 2 to EUTRAN_SECURITY_ENABLED.